specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.0/examples/GetCertificatePolicy-example.json

```
@@ -22,7 +22,8 @@
           "key_usage": [],
           "validity_months": 297,
           "basic_constraints": {
-            "ca": false
+            "ca": false,
+            "description": "Optional for leaf certs but needs to be true for CA certificate, if building PKI on top of key vault."
```

Should capitalize "Key Vault" when used as a product name:
```suggestion
            "description": "Optional for leaf certs but needs to be true for CA certificate, if building PKI on top of Key Vault."
```